CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
 * Improve performance of rolling back write transactions after making changes. If no KVO observers are used this is now constant time rather than taking time proportional to the number of changes to be rolled back. Rollbacks with KVO observers are 10-20% faster. ([PR #6513](https://github.com/realm/realm-core/pull/6513))
+* Enable PBS to FLX Migration feature that will migrate the local realm to use FLX if the server has been migrated to FLX. ([#6342](https://github.com/realm/realm-core/issues/6342))
```

Comment:
I think this entry should be about the feature itself (not about enabling a feature) and also a proper link to the feature

Reply:
Unfortunately, there isn't an issue specifically for the feature, since github issues aren't created for epics. I can update the comment to reflect the feature, rather than just enabling it..